proposals/0004-open-api.md

```
@@ -0,0 +1,787 @@
+
+Note that for some fields, a type reference is specified, which will be used to automatically supply the schema definition in the OpenAPI specification.  This is an optional part of the definition, where in cases when it is unspecified, a default schema of `any` will be used.
+
+Because KDoc links do not support optional modifiers or generics, we can use a custom syntax for indicating arrays, maps, and optionals.
+
+| Modifier | Format           | JSON Schema mapping                                                        |
+|----------|------------------|----------------------------------------------------------------------------|
+| `+`      | `[String]+`      | [Array](https://json-schema.org/understanding-json-schema/reference/array) |
+| `?`      | `[String]?`      | [Required](https://www.learnjsonschema.com/2020-12/validation/required/)                                                               |
+| `:`      | `[String]:[Any]` | M
+
+# Summary
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content

(MD024, no-duplicate-heading)